lib/SPIRV/SPIRVRegularizeLLVM.cpp

```diff
@@ -563,6 +582,19 @@ bool SPIRVRegularizeLLVMBase::regularize() {
           }
         }
 
+        if (II.isLogicalShift() &&
```

Comment:
Please add some high-level comment here about why are we doing this regularization.

Reply:
Thanks, all done in [343c988](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1450/commits/343c9888c8e3b8e57046e51d364769047367fb13)